pkg/cvo/upgradeable.go

```diff
@@ -215,9 +216,26 @@ func (check *clusterVersionOverridesUpgradebale) Check() *configv1.ClusterOperat
 	return cond
 }
 
+type clusterManifestExcludesUpgradable struct {
+	filter payload.RegexpFilter
+}
+
+func (check *clusterManifestExcludesUpgradable) Check() *configv1.ClusterOperatorStatusCondition {
+	if len(check.filter) == 0 {
+		return nil
+	}
+	return &configv1.ClusterOperatorStatusCondition{
+		Type:    configv1.ClusterStatusConditionType("UpgradableManifestExcludes"),
+		Status:  configv1.ConditionFalse,
+		Reason:  "ManifestExcludesSet",
+		Message: "Certain manifests are excluded from the operator's payload. The cluster cannot be upgraded.",
```

Comment:
> https://github.com/openshift/cluster-version-operator/pull/252/files#diff-d9212d08828a9f9d8b54a2e90f31500aR215

based on that, something like `Excluding manifests from the release image prevents upgrades. Please remove the exclude filters before continuing.`

Reply:
I don't like regex on manifests, I'd like this to be a higher level name that manifests are associated with.  If that's the case, then this shouldn't even be a condition because expectation is that they are tested and we don't need this alert.